pyarn/lexer.py

```diff
@@ -35,6 +35,11 @@ def t_NUMBER(t):
     return t
 
 
+# Unquoted string regex (see t_STRING)
+# Docstrings cannot use string interpolation, this value is used by other modules
+UNQUOTED_STRING = r'[a-zA-Z/.-]([^\s\n,]*[^\s\n,:])?'
```

Comment:
Just out of curiosity, doesn't black prefer double quotes over single?

Reply:
It does, but this project does not (yet?) use `black` and most of the strings I've seen here use single quotes

Reply:
Yep. that's why question out of curiosity. Perhaps in the future we could just start using it which will fix it all.

Reply:
I did not bring black in for the first commits intentionally. I thought pyarn was quite small and I did not see any need to enforce styles in the sense black does it. +1 to bring it in if you feel it would be worth it.

Reply:
Why is this defined in the lexer and not in `lockfile.py` itself? ?

Reply:
> Why is this defined in the lexer and not in `lockfile.py` itself? ?

I wanted this to be defined as close to `t_STRING` as possible in the hope that if someone changes the relevant part of the `t_STRING` regex, they will remember to update the `UNQUOTED_STRING` regex accordingly (it is just copy-pasted from there)

Reply:
Sounds good!